cli/integration/tests/waiter/util.py

```diff
@@ -403,3 +403,24 @@ def get_instances_not_in_output(possible_instances, stdout_output):
     return [instance
             for instance in possible_instances
             if instance['id'] not in stdout_output]
+
+def specific_instances_for_service(waiter_url, service_id, instance_type, expected_status_code=200):
+
+    """returns specific instances map of a service"""
+    headers = {
+        'Content-Type': 'application/json',
+        'x-cid': cid()
+    }
+    response = session.get(f'{waiter_url}/apps/{service_id}', headers=headers)
+    service = response.json()
+    assert expected_status_code == response.status_code, \
+        f'Expected {expected_status_code}, got {response.status_code} with body {response.text}'
+    if instance_type == 'active-instances':
+        return service['instances']['active-instances']
+    elif instance_type == 'killed-instances':
+        return service['instances']['killed-instances']
+    elif instance_type == 'failed-instances':
+        return service['instances']['failed-instances']
```

Comment:
Should we return service['instances'][instance_type] and avoid if-else branching?